pvlib/tests/ivtools/test_sdm.py

```diff
@@ -379,4 +380,23 @@ def test__calc_theta_phi_exact_vector():
     assert np.isnan(theta[0])
     assert np.isnan(theta[1])
     assert np.isnan(phi[0])
-    np.testing.assert_allclose(phi[1], 2.2079, atol=.0001)
+    assert_allclose(phi[1], 2.2079, atol=.0001)
+
+
+def test_pvsyst_temperature_coeff():
+    # test for consistency with dP/dT estimated with secant rule
+    params = {'alpha_sc': 0., 'gamma_ref': 1.1, 'mu_gamma': 0.,
+              'I_L_ref': 6., 'I_o_ref': 5.e-9, 'R_sh_ref': 200.,
+              'R_sh_0': 2000., 'R_s': 0.5, 'cells_in_series': 60}
+    expected = -0.7489980887568493
```

Comment:
Seems a bit large for a realistic module. I don't have enough intuition with the params values to say more. I just want to double check that this really is the expected value and the function really is returning the right thing.

Reply:
Good catch. The value was in W/C. Fixed now.